assets/components/cards/render/CardMeta.jsx

```diff
@@ -1,42 +1,32 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import {gettext, shortDate, fullDate, isDisplayed} from 'utils';
+import {characterCount, wordCount} from 'utils';
+import WireListItemIcons from 'wire/components/WireListItemIcons';
 
-
-function CardMeta({wordCount, charCount, pictureAvailable, source, versioncreated, displayDivider, slugline, listConfig}) {
+function CardMeta({item, pictureAvailable, displayDivider, slugline, listConfig}) {
```

Comment:
Should `pictureAvailable` be changed to `picture` (I see props being passed to `CardMeta` as `picture`)?

Reply:
ups :man_facepalming: